src/app/components/publiclists/publiclists.component.spec.ts

```diff
@@ -28,6 +33,15 @@ const routes: Routes = [
       {path: 'me', component: ChallengelistComponent}
     ]
   },
+  {
```

Comment:
@sanketbansal, If I'm correct, these changes are from an earlier PR, right?

Reply:
yeah I think he worked on top of all his previous PR's and also pushed the previous code as well in all the PR's, that's why in all the PR's it about 35K lines of code